src/services/bulkstatus.service.js

```diff
@@ -109,6 +109,28 @@ async function checkBulkStatus(req, res) {
       ],
       extension: { 'https://example.com/extra-property': true }
     };
+
+    if (bulkStatus.failedOutcomes.length > 0) {
+      bulkStatus.failedOutcomes.forEach(fail => {
+        const failOutcome = {};
+        failOutcome.id = uuidv4();
+        failOutcome.issue = [
+          {
+            severity: 'error',
+            code: 'BadRequest',
```

Comment:
Fine to force this to a BadRequest here? I think this can only happen from bad input from the export server, but not sure if that qualifies as a bad request